data-science-onramp/feature-engineering/feature_engineering.ipynb

```diff
@@ -1,4795 +1,1019 @@
 {
```

Comment:
<p>Original:</p><p>Run the following cell to load the Gas Prices dataset from BigQuery into a dataframe. It will define a query to select the columns needed from the gas prices dataset, run the query using the BigQuery client, and then convert it to a Pandas DataFrame. </p><p> </p><p>Proposed:</p><p>Run the following cell to load the Gas Prices dataset from BigQuery into a dataframe. <strong>You</strong> will define a query <strong>that</strong> <strong>selects</strong> the columns needed from the gas prices dataset, run the query using the BigQuery client, and then convert it to a Pandas DataFrame. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/leahecole/python-docs-samples/pull/10/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>

Reply:
<p>Done</p>